ksonnet/monitoring/prototypes/all.jsonnet

```diff
@@ -4,7 +4,6 @@
 // @param name string Name to give to each of the components
 // @shortDescription monitoring components. This create user environment.
 // @optionalParam dkubeDockerSecret string  Docker secret for dkube images
-// @optionalParam dkubeD3extImage string ocdr/dkube-d3ext:alpha3 Dkube d3ext image
```

Comment:
Why remove this? Where are we accepting the image?

Reply:
we are accepting dkubeD3extImage in dkube package. We moved dkube-d3ext to dkube from monitoring component. So no need to put parameter here